src/Params/WaveTable.h

```diff
@@ -222,6 +230,11 @@ class Tensor<1, T> : public TensorBase<1, T>
     bool operator!=(const Tensor<1, T>& other) const {
         return !operator==(other); }
 
+    void fillWithZeroes()
+    {
+        std::fill_n(m_data, base_type::size(), 0);
```

Comment:
Usually I stick with memset, though the compiler may optimize this call to precisely that.

Reply:
This code *might* be removed soon. I leave the comment open.